ui/src/components/SignoffSummary/index.jsx

```diff
@@ -46,24 +61,83 @@ function SignoffSummary(props) {
             Requires Signoffs From
           </ListSubheader>
         }>
-        {Object.entries(requiredSignoffs).map(([role, count], index) => {
+        {theRequiredSignoffs.map(([role, count], index) => {
           const key = `${role}-${index}`;
+          // AllSigned Returns all that have signed
+          let AllSigned = [];
+
+          if (listOfSignoffs) {
+            AllSigned = listOfSignoffs.filter(arr => {
+              return role === arr[1];
+            });
+          }
+
+          // AllNotSigned() Gets and returns all that haven't signed
+          const AllNotSigned = () => {
+            const leftarr = [];
+            const allLeft = count - AllSigned.length;
+
+            // Disabled eslint because "i+1" runs loop till eternity
+            // and for in loop only works on arrays
+            // eslint-disable-next-line no-plusplus
```

Comment:
Seems reasonable to disable this given that [eslint has a carve out explicitly for usage in a for loop](https://eslint.org/docs/latest/rules/no-plusplus#allowforloopafterthoughts).